keps/sig-windows/1981-windows-privileged-container-support/README.md

```diff
@@ -718,6 +762,83 @@ Beta
 - Validate behaviors of various volume mount types as described in [Container Mounts](#container-mounts) with e2e tests
 - Add e2e tests to test different ways to construct paths for container command, args, and workingDir fields for both `hostProcess` and non-hostProcess containers. These tests will include constructing paths with and without `$CONTAINER_SANDBOX_MOUNT_POINT` set and with different combinations of forward and backward slashes.
 
+Graduation
+
+- Add e2e tests to validate running `hostProcess` containers as non SYSTEM/admin accounts
+- Update e2e tests for new volume mount behavior as desdribed in [Container Mounts](#container-mounts)
```

Comment:
```suggestion
- Update e2e tests for new volume mount behavior as described in [Container Mounts](#container-mounts)
```

since bind mount is not yet released in containerd 1.7, do we even have the ability to add tests for this yet?

Reply:
Yes we do.

We have a github action that runs every night that builds containerd and hcsshim from `main` and publishes the package to https://github.com/kubernetes-sigs/sig-windows-tools/releases/tag/windows-containerd-nightly.
This package already has the changes to use the new `bind` mount behavior (if running on Windows Server 2022 for now, it will work on Windows Server 2019 in a few weeks).

We use this package in https://testgrid.k8s.io/sig-windows-signal#capz-windows-containerd-nightly-master (and a few others).

My plans were to update the e2e tests to check for the contaienrd version being used on the nodes and add skips to the e2e tests that require a different version of containerd.